sklearn/feature_extraction/image.py

```diff
@@ -139,6 +138,14 @@ def _to_graph(
     return return_as(graph)
 
 
+@validate_params(
+    {
+        "img": ["array-like"],  # Could it also be "sparse matrix"?
```

Comment:
```suggestion
        "img": ["array-like"],
```

This cannot be a sparse matrix because `np.atleast_3d` will not work as expected on a sparse matrix.

However, we should update the documentation because it can be a list of list of list which is indeed an `array-like`.